doc/coding_standard.md

```diff
@@ -130,12 +130,16 @@ call();
 ```
 
 For [doxygen](https://www.doxygen.nl/) style comments `/** */` should be used.
+For single-line doxygen-style comment, use `///`.
```

Comment:
Do you want to mandate `///` or just sugest it as an acceptable alternative for one-liner ?

Reply:
I let you resolve this whichever is best. I will approve right away.

Reply:
I think that at this point in time we should allow either `/** text */` or `/// text` for single line comments just based on the sheer number of `/** text */` comments that exist in the code. If we move to only `///` we would then need to undertake the massive effort of modifying those existing comments.